src/utils/helpers/list/mapColumnsByRole.ts

```diff
@@ -89,14 +96,19 @@ export const useMapColumnsByRole = () => {
       sortName: 'outcome',
       ColumnComponent: OutcomeCell,
     },
+    ASSIGNER_ACTION: {
+      headerName: strings.COLUMN_ASSIGNER_ACTION,
+      sortName: 'assigner-action',
+      ColumnComponent: AssignerActionCell,
+    },
     REVIEWER_ACTION: {
       headerName: strings.COLUMN_REVIEWER_ACTION,
-      sortName: 'outcome',
+      sortName: 'reviewer-action',
       ColumnComponent: ReviewerActionCell,
     },
     APPLICANT_ACTION: {
       headerName: strings.COLUMN_APPLICANT_ACTION,
-      sortName: 'outcome',
+      sortName: 'applicant-action',
```

Comment:
I think this might be the problem. We don't want to change the sorting. The "Reviewer action" column (etc.). shouldn't be sortable.